### PR TITLE
Major Organization

### PR DESCRIPTION
**Pull request by BenjaminHsieh**
Created new directories in utils: functions and graphing, I have already updated the tests and script paths according, please double check let me know if there are errors

Created new directory in paper: data_results: please save future data results we wish to report in paper here (for example those txt files that are important), accordingly please update scripts saving locations

Changes some coverage issues and updated makefiles

Deleted old-paper respository: we dont need anything from here right?
I did most of 1,2,3 from #114 , except updating scripts to save txt files

Please pull and update any scripts accordingly, let me know anything wrong

### CODE REVIEW

**Comment by briankleinqiu** [comment]
Structure looks good. Lets pull this in as soon as we can

File: code/utils/tests/test_lme_functions.py [hunk 1]
```diff
@@ -12,14 +12,14 @@
...
-import sys
+import sys, os
 from scipy import stats
 from sklearn import linear_model
 from numpy.testing import assert_almost_equal, assert_allclose
 
 
 # Append function path
-sys.path.append('..')
+sys.path.append(os.path.join(os.path.dirname(__file__), "../functions/"))
```
Comment (briankleinqiu) [inline code/utils/tests/test_lme_functions.py:22]: out of curiosity what advantage does using os.path give

**Comment by BenjaminHsieh** [comment]
not sure, but the other way fails when I run make test

**Comment by BenjaminHsieh** [comment]
yeah trying to figure out the travis

**Comment by changsiyao** [comment]
In test_plot_roc, t_roc needs to be float? Right now, its rounding everything to integer. Like, try " t_roc = np.array([[1.0 \* 2/4, 1.0 \* 1/4], [1.0 \* 1/6, 1.0 \* 1/2], [1.0 \* 1/4, 1.0 \* 2/5], [1.0 \* 2/3, 1.0 \* 4/5]]) "?

**Comment by BenjaminHsieh** [comment]
I imported division from the future, so that shouldn't matter, will change it anyway

**Comment by BenjaminHsieh** [comment]
Travis passed! lots of changes to structure and tests, should merge asap

**Comment by BenjaminHsieh** [comment]
Please merge @brianqiu @changsiyao @boyinggong @pigriver123

**Pull request merged**